hnn_core/mpi_child.py

```diff
@@ -0,0 +1,73 @@
+"""Script for running parallel simulations with MPI"""
```

Comment:
where is this script used?

Reply:
https://github.com/jonescompneurolab/hnn-core/blob/1cc993d6289ceb30b317a86813f4190a8205fddf/hnn_core/mpi_child.py#L1-L3